src/modules/UI/scenes/EditToken/EditToken.ui.js

```diff
@@ -14,22 +14,38 @@ import styles from './style.js'
 import {PrimaryButton, TertiaryButton} from '../../components/Buttons'
 import {FormField} from '../../../../components/FormField.js'
 // import * as WALLET_ACTIONS from '../../Wallets/action.js'
-import type {CustomTokenInfo} from '../../../../types.js'
 import StylizedModal from '../../components/Modal/Modal.ui'
 import DeleteTokenButtons from './components/DeleteTokenButtons.ui.js'
 import * as Constants from '../../../../constants/indexConstants'
 import OptionIcon from '../../components/OptionIcon/OptionIcon.ui'
 import * as UTILS from '../../../utils'
 import type {AbcMetaToken} from 'airbitz-core-types'
+import type {CustomTokenInfo} from '../../../../types.js'
+
 import _ from 'lodash'
 
-export type DispatchProps = {
+export type EditTokenDispatchProps = {
   showDeleteTokenModal: () => void,
   hideDeleteTokenModal: () => void,
   deleteCustomToken: (string, string) => void,
   editCustomToken: (string, string, string, string, string, string) => void
```

Comment:
If you come across functions that have unnamed parameters in their type signature, please add names
`(string, string, string, string) => {...}` ---> `(walletId: string, walletName: string, balance: string) => {...}`